RevenueCatUI/Runtime/Platforms/Android/AndroidCustomerCenterPresenter.cs

```diff
@@ -0,0 +1,42 @@
+#if UNITY_ANDROID && !UNITY_EDITOR
+using System;
+using System.Threading.Tasks;
+using UnityEngine;
+using RevenueCat.UI.Internal;
+
+namespace RevenueCat.UI.Platforms
+{
+    internal class AndroidCustomerCenterPresenter : ICustomerCenterPresenter
+    {
+        private readonly AndroidJavaClass _plugin;
+        private TaskCompletionSource<bool> _current;
+
+        public AndroidCustomerCenterPresenter()
+        {
+            _plugin = new AndroidJavaClass("com.revenuecat.unity.RevenueCatUI");
+            RevenueCatUICallbackHandler.Initialize();
+            RevenueCatUICallbackHandler.SetAndroidCustomerCenterPresenter(this);
+        }
+
+        public bool IsSupported()
+        {
+            try { return _plugin.CallStatic<bool>("isSupported"); }
+            catch { return false; }
```

Comment:
Might be worth adding a debug log in this catch? Also the same in the other method